### DISCUSSION TITLE
Welcome to Free-courses-with-Certificates Discussions!

### DISCUSSION THREAD

**Discussion by avinash201199** [avinash201199/Free-courses-with-Certificates#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by avinash201199** [comment]
Post yout query here :)

**Comment by Pavan-Rajesh** [comment]
I want to integrate supabase into this project for authentication and database can I do it?

**Reply by avinash201199** [reply]
you can
